modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/bloodsucker.dm

```diff
@@ -429,13 +429,11 @@
 	//Traits: Species
 	if(ishuman(owner.current))
 		var/mob/living/carbon/human/user = owner.current
-		var/obj/item/bodypart/user_left_arm = user.get_bodypart(BODY_ZONE_L_ARM)
-		var/obj/item/bodypart/user_right_arm = user.get_bodypart(BODY_ZONE_R_ARM)
-		user.dna?.remove_all_mutations()
```

Comment:
noticed you removed user.dna?.remove_all_mutations, but it makes sense to not have anyway

Reply:
whoops